content/docs/components/training/mpi.md

```diff
@@ -6,9 +6,19 @@ weight = 25
 
 This guide walks you through using MPI for training.
 
+The MPI Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out the list of adopters [here](https://github.com/kubeflow/mpi-operator/blob/master/ADOPTERS.md).
+
 ## Installation
 
-If you haven’t already done so please follow the [Getting Started Guide](https://www.kubeflow.org/docs/started/getting-started/) to deploy Kubeflow.
+You can deploy the operator with default settings without using `kustomize` by running the following commands:
+
+```shell
+git clone https://github.com/kubeflow/mpi-operator
+cd mpi-operator
+kubectl create -f deploy/mpi-operator.yaml
+```
+
+Alternatively, if you haven’t already done so please follow the [Getting Started Guide](https://www.kubeflow.org/docs/started/getting-started/) to deploy Kubeflow.
```

Comment:
Some more best practice tips here: Avoid capital letters or any assumption that page titles will remain the same for ever :). We use relative URLs for all links within the docs, because absolute links break the doc versioning. And the words "if you haven't already done so" may confuse readers because they're not sure if they need to follow the getting-started guide after doing the cloning. And lastly :) we avoid the word "please" because sometimes it gives a different impression from the one we intend. How about this:

```
Alternatively, follow the [getting started guide](/docs/started/getting-started/) to deploy Kubeflow.
```